easse/cli.py

```diff
@@ -26,9 +53,17 @@ def common_options(function):
             help='test set to use.',
     )(function)
     function = click.option(
-            '--input_path', '-i', type=click.Path(), default=None,
+            '--sys_sents_path', type=click.Path(), default=None,
             help='Path to the system predictions input file that is to be evaluated.',
     )(function)
+    function = click.option(
+            '--orig_sents_path', type=click.Path(), default=None,
+            help='Path to the source sentences. Only used when test_set == "custom"',
+    )(function)
+    function = click.option(
+            '--refs_sents_paths', type=str, default=None,
+            help='Comma-separated list of path(s) to the references(s). Only used when test_set == "custom"',
```

Comment:
A comma-separated list is pretty convenient for a first version. But, couldn't we use * in a similar was as [SacreBleu](https://github.com/mjpost/sacreBLEU/blob/1ab268494b2aad6830231aa7d218d217a5e3749e/sacrebleu.py#L1490)?

Reply:
It is sadly not available in Click by default I think:
https://stackoverflow.com/questions/48391777/nargs-equivalent-for-options-in-click